lib/client.js

```diff
@@ -216,8 +217,20 @@ exports = module.exports = class Tgfancy extends TelegramBot {
 
         // We are now performing 'on ready' actions.
         // Let's set the correct webhook when on Openshift
-        if (this.options._openshift) {
+        if (this.options._openshift && process.env.OPENSHIFT_APP_UUID) {
             this.setWebHook(this.options._openshiftDomain);
+
+        // setting up listening to updates via websocket
+        } else if (this.options.webSocket) {
+            if (options.polling || options.webHook) {
```

Comment:
Or maybe just exactly that, 
```
if (this.options.webSocket) {
  if (options.polling || options.webHook) {
    throw new Error
  }
}
```

After all, you mentioned working on only allowing webhook or polling at one time in n-t-b-a, so I think we don't have to cover that in this repo too